test/views/ViewArrivalsSpec.scala

```diff
@@ -75,9 +77,48 @@ class ViewArrivalsSpec extends SingleViewSpec("viewArrivals.njk") with Generator
     ls.eq(5).text() mustBe "11 August 2020"
   }
 
+  val rows: Elements = doc.getElementsByAttributeValue("role", "row")
+
   "generate a row for each arrival" in {
-    val ls: Elements = doc.getElementsByAttributeValue("role", "row")
+    rows.size() mustEqual 7
+  }
 
-    ls.size() mustEqual 7
+  "display rows in correct (sorted) order" in {
```

Comment:
The view doesn't seem to do the sorting just displays the elements in the same order as it is given in the view data, it think removing the "sorted" word should make it clear that there isn't any logic here for that. What about phrasing it as 
```suggestion
  "display rows in the same order as the view data" in {
```